src/codegeneration/TempVarTransformer.js

```diff
@@ -110,13 +113,15 @@ class VarScope {
 export class TempVarTransformer extends ParseTreeTransformer {
   /**
    * @param {UniqueIdentifierGenerator} identifierGenerator
+   * @param {Options} options
    */
-  constructor(identifierGenerator) {
+  constructor(identifierGenerator, options) {
```

Comment:
I guess the other way to go here is to pass `reporter` in to `TempVarTransformer` so all of the transformers use the same constructor parameters. That way we don't need the dummy constructors that only shorten the arg list. The unused reporter argument is common to many transformers, they don't emit errors at this time.